src/main/include/utils/CommandUtils.h

```diff
@@ -110,7 +112,7 @@ static frc2::CommandPtr ScoreShoot(std::function<ScoringDirection()> direction,
                                 "Scoring Composition", "fed %s", "");
                           }).ToPtr())
                  .AndThen(frc2::WaitCommand(kFlywheelRampDelay).ToPtr())
-                  // .AndThen(PreScoreShuffle(direction, scoring, intake))
+                 // .AndThen(PreScoreShuffle(direction, scoring, intake))
                  .AndThen(Shoot(intake, scoring, direction).ToPtr())
                  .AndThen(frc2::InstantCommand([] {
```

Comment:
Should the `ScoreShoot` command always move the arm back to 10 degrees even if it's interrupted early? Might prevent cases where it gets stuck in the up position.

Reply:
I believe the goal was to add this to the intake command also, so even if it got stuck up it could be returned simply by running intake.